app/services/imports/user_import_service.rb

```diff
@@ -26,19 +26,20 @@ def create_user(xml_document)
         role = highest_role(user.role, role(user_field_value(xml_document, "user-type")))
         user.update!(role:, is_dpo:)
         user.legacy_users.create!(old_user_id:)
-        @logger.info("Found duplicated email, updating user #{user.id} with role #{role} and is_dpo #{is_dpo}")
       else
-        user = User.create!(
-          email:,
-          name:,
-          password: Devise.friendly_token,
-          phone: user_field_value(xml_document, "telephone-no"),
-          organisation:,
-          role: role(user_field_value(xml_document, "user-type")),
-          is_dpo: is_dpo?(user_field_value(xml_document, "user-type")),
-          is_key_contact: is_key_contact?(user_field_value(xml_document, "contact-priority-id")),
-          active: user_field_value(xml_document, "active"),
-        )
+        user = User.new
```

Comment:
is this an efficiency improvement?

Reply:
It shouldn't be, but it lets you tell devise to skip the confirmation notification before attempting to save the user - we then trigger them at the end of the import, once all the logs are in (but we need the user objects to exist earlier).